.travis.yml

```diff
@@ -0,0 +1,63 @@
+language: python
+
+cache:
+  pip: true
+  apt: true
+
+sudo: false
+dist: trusty
+
+
+matrix:
+  include:
+    - env: TEST=standard
+      os: linux
+      python: '2.7'
+    - env: TEST=standard
+      os: linux
+      python: '3.6'
+
+
+before_install:
+    # Instal miniconda
+    - if [ "${TRAVIS_OS_NAME}" == "linux" ]; then
+        wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+      else
+        wget http://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O miniconda.sh;
+      fi;
+    - bash miniconda.sh -b -p $HOME/miniconda
+    - export PATH="$HOME/miniconda/bin:$PATH"
+    - hash -r
+    - conda config --set always_yes yes --set changeps1 no
+    - conda update -q conda
+    - conda info -a
+    - SRC_DIR=$(pwd)
+
+
+install:
+    # Create the python environment ;
+    - conda create -q -n testenv python=$TRAVIS_PYTHON_VERSION numpy pip cython matplotlib;
```

Comment:
why cython?

Reply:
yes, my first installation was without cython, but it seems to be needed for mkl. See [this](https://travis-ci.org/EtienneCmb/ephypype/jobs/424025003#L811)

Reply:
I don't see a directory for docs. I don't know how the docs are being built. It seems the docs are common for all 3 repos. So I don't know how you would do CircleCI here ...

Reply:
Right. Doc is in a separate [directory](https://github.com/neuropycon/neuropycon_doc)

Reply:
yep. Let's tackle easy stuff first :-)

Reply:
@jasmainak do you have experience with mne data? If so, may be you can increment this PR to download mne-data and cache it?

Reply:
yep, let me try ...